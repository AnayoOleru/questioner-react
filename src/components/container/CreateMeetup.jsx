import React, { Component, Fragment } from 'react';
import Header from '../presentation/Header';
import Footer from '../presentation/Footer';

class CreateMeetup extends Component {
  render() {
    return (
      <Fragment>
        <Header />
        <div className='CreateMeetup'>
          <div className='create-meetup-area'>
            <div className='container'>
              <div className='create-meetup-form-area'>
                <h3 className='create-meetup-form-text'>Create Meetup</h3>
                <form>
                  <div className='create-meetup-input-section'>
                    <input
                      type='text'
                      placeholder='Name'
                      required
                    />
                  </div>
                  <div className='create-meetup-input-section'>
                    <textarea required placeholder='Description' l />
                  </div>
                  <div className='create-meetup-form-submit'>
                    <button type='submit'>Submit</button>
                  </div>
                </form>
              </div>
            </div>
          </div>
        </div>
        <Footer />
      </Fragment>
    );
  }
}
export default CreateMeetup;
