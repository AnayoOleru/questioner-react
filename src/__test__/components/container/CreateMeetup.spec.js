import React from 'react';
import { shallow } from 'enzyme';
import CreateMeetup from '../../../components/container/CreateMeetup';

describe('Create meetup Component', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<CreateMeetup />);
  });
  test('Create meetup component should render', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
