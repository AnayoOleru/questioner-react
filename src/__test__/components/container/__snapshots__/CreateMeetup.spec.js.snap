// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Create meetup Component Create meetup component should render 1`] = `ShallowWrapper {}`;
